File: src/main/java/com/sudosoo/takeiteasy/dto/heart/PostHeartRequestDto.java
package com.sudosoo.takeiteasy.dto.heart;

import jakarta.validation.constraints.NotNull;
import lombok.*;

@Getter
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor
public class PostHeartRequestDto {
    @NotNull(message = "유저아이디를 입력해 주세요.")
    private Long memberId;
    @NotNull(message = "게시글아이디를 입력해 주세요.")
    private Long postId;
}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/dto/heart/PostHeartRequestDto.kt
package com.sudosoo.takeItEasy.dto.heart

import jakarta.validation.constraints.NotBlank
class PostHeartRequestDto(
    @NotBlank(message = "유저아이디를 입력해 주세요.")
    val memberId: Long,
    @NotBlank(message = "게시글아이디를 입력해 주세요.")
    val postId: Long
)